examples/bmark_ampl.jl

```diff
@@ -0,0 +1,23 @@
+using JLD2, Plots
+using AmplNLReader, Percival, NLPModelsIpopt, SolverBenchmark
+
+# define problems
+problem_names = readdir(joinpath(@__DIR__, "ampl"))
+problems = (AmplModel(joinpath(@__DIR__, "ampl", p)) for p in problem_names)
+
+# define solvers
+solvers = Dict(
+  :percival => nlp -> percival(nlp, atol = 1.0e-4, rtol = 1.0e-5, max_time = 10.0),
+  :ipopt => nlp -> ipopt(nlp, tol = 1.0e-5, print_level = 0, sb = "no"),
```

Comment:
What is `sb = no`?

Maybe add a `max_cpu_time = 10.0` for Ipopt and `verbose = 0` for Percival, so that the options "match".

Reply:
`sb = 0` prevents IPOPT from displaying the banner every time.